src/qref/schema_v1.py

```diff
@@ -55,6 +56,35 @@ def _inner(v):
 _source_sorter = AfterValidator(_sorter(lambda c: c.source))
 
 
+def _parse_connection(connection):
+    if isinstance(connection, str):
+        source, target = connection.replace(" ", "").split("->")
+        return {"source": source, "target": target}
+    return connection
+
+
+_connection_parser = BeforeValidator(_parse_connection)
+
+
+CONNECTION_SCHEMA = {
+    "type": "array",
+    "items": {
+        "anyOf": [
+            {"$ref": "#/$defs/Connection"},
+            {
+                "pattern": (
+                    "^([A-Za-z_][A-Za-z0-9_]*\\.)*[A-Za-z_][A-Za-z0-9_]* -> "
```

Comment:
Why you're not using `CONNECTON_PATTERN` here?

Reply:
This part was automatically generated, and I wasn't altering it because Javascript regular expressions slightly differ from the Python ones. However, this particular pattern is the same in both languages, so I agree it's best to change it to `CONNECTION_PATTERN`. Fixed!